src/auth-service/utils/join.js

```diff
@@ -140,12 +140,18 @@ const join = {
             (value) => {
               if (value.success == false) {
                 const errors = value.errors ? value.errors : "";
-                callback({
+                logObject("the validation checks results", {
```

Comment:
addressing the mailing server error. 
The tool we used was throwing 403 because our credit ($$) on it is down.